.circleci/config.yml

```diff
@@ -164,7 +164,7 @@ parameters:
     default: "main"
     type: string
   sandbox_git_branch:  # change to feature branch to test deployment
-    default: "add-wysisyg-editor"
+    default: "kw-fix-piv-card-login"
```

Comment:
Not sure why this was changed. Did we loose what ever purpose the previous value served?